recipe/meta.yaml

```
@@ -66,20 +73,24 @@ test:
         - celery.utils.dispatch
         - celery.worker
     commands:
-        - celery --version
-        - celery worker --help
-        - celery beat --help
-        - celery multi help
-        - celery shell --help
-        - pytest t/unit  # [py3k]
+        # Running test is limited because it requires 'pytest=8.3.5' which is not available now
+        - pip check
+        - pytest -vv t/unit {{ u_test_files_to_skip }} -k "not ({{ u_tests_to_skip }})" # [py<312]
```

When I ran it I was wondering why we weren't running any tests at all 🤔 

A feature of conda build is that it runs the python versions in a random order from run to run so it's hard to follow and I was watching the pythons >= 3.12.

However, your selector, here, `[py<312]` says run `pytest + args` on python less than 3.12 which is great.

But there's no `pytest` line for version of python `>=3.12`.

So, depending on how exciting the tests get you'll also want:

```suggestion
        - pytest -vv t/unit {{ u_test_files_to_skip }} -k "not ({{ u_tests_to_skip }})" # [py<312]
        - pytest -vv t/unit" # [py>=312]
```

(I would also add enough SPACE chars to make the comments line up which is too hard to do in a GH comment box 😆 )